src/app/(app)/home/entries/[entryId]/page.jsx

```diff
@@ -8,12 +8,19 @@ export default async function Page({ params }) {
   return (
     <section className={styles.viewerContainer}>
       <div className="viewer-nav">
-        {/* <Link href="/home">back to home</Link> */}
+        {/* <Link href="/home">back to list</Link> */}
       </div>
       
       <section className={styles.viewerDetails}>
-          <span>{entry[0].created_at.toLocaleDateString('en-us', { weekday:"long", year:"numeric", month:"short", day:"numeric"})}</span>
-        </section>
+          <div className={styles.createdTimeDate}>
+            <span className={styles.date}>{entry[0].created_at.toLocaleDateString('en-us', { weekday:"long", year:"numeric", month:"short", day:"numeric"})}</span>|
+            <span className={styles.date}>{entry[0].created_at.toLocaleTimeString('en-us', { timeStyle: 'short' })}</span>
+          </div>
+          <div className={styles.edit}>
+            <button className={styles.editButton}>Edit</button>
```

Comment:
NIT: Old habits, but could you change `Edit` to `edit` and capitalize using styles? For markdown we want to let our CSS handle formatting opinions